bugsnag-android-core/src/main/java/com/bugsnag/android/EventStore.java

```diff
@@ -59,7 +59,7 @@ public int compare(File lhs, File rhs) {
         this.delegate = delegate;
     }
 
-    void flushOnLaunch() {
```

Comment:
Making Notifier a member variable would avoid some of the method signature changes here too.